src/tests/userprofile.test.js

```diff
@@ -40,6 +42,68 @@ describe('GET /api/v1/profiles', () => {
   });
 });
 
+describe('GET /api/v1/profiles', () => {
+  it('Should return an error message if user tries to fetch user profiles before they log in', (done) => {
+    chai.request(app)
+      .get('/api/v1/profiles')
+      .end((err, res) => {
+        if (err) done(err);
+        expect(res).to.have.status(401);
+        expect(res.body).to.be.an('object');
+        expect(res.body).to.have.keys('error');
+        expect(res.body.error).to.deep.equal('Please log in or Register');
+        done();
+      });
+  });
+});
+
+describe('GET /api/v1/profiles', () => {
+  it('Should A list of profiles if  user is identified', (done) => {
+    chai.request(app)
+      .get('/api/v1/profiles')
+      .set('Authorization', userToken)
+      .end((err, res) => {
+        if (err) done(err);
+        expect(res).to.have.status(200);
+        expect(res.body).to.be.an('object');
+        expect(res.body).to.have.keys('message', 'data');
+        expect(res.body.data[0]).to.have.keys('userName', 'bio', 'image');
+        done();
+      });
+  });
+});
+
+describe('GET /api/v1/profiles', () => {
+  it('Should return an error message if user tries to fetch user profiles before they log in', (done) => {
+    chai.request(app)
+      .get('/api/v1/profiles')
+      .end((err, res) => {
+        if (err) done(err);
+        expect(res).to.have.status(401);
+        expect(res.body).to.be.an('object');
+        expect(res.body).to.have.keys('error');
+        expect(res.body.error).to.deep.equal('Please log in or Register');
+        done();
+      });
+  });
+});
+
+describe('GET /api/v1/profiles', () => {
```

Comment:
this looks like a copy of https://github.com/andela/coding-geeks-ah-backend/pull/19/files#diff-7e80fd915c1e3097f4adf33a8a3c7f0dR60

Reply:
I'd also like to see an assertion about the number of profiles returned is what we expect

Reply:
> this looks like a copy of https://github.com/andela/coding-geeks-ah-backend/pull/19/files#diff-7e80fd915c1e3097f4adf33a8a3c7f0dR60

Ok...fixed

Reply:
> I'd also like to see an assertion about the number of profiles returned is what we expect

On it...